README.md

```diff
@@ -1226,6 +1226,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
 
 * [autopilot-rs/autopilot-rs](https://github.com/autopilot-rs/autopilot-rs) — A simple, cross-platform GUI automation library for Rust.
 * [maps4print/azul](https://github.com/maps4print/azul) — A free, functional, IMGUI-oriented GUI framework for rapid development of desktop applications written in Rust, supported by the Mozilla WebRender rendering engine. [<img src="https://api.travis-ci.org/maps4print/azul.svg?branch=master">](https://travis-ci.org/maps4print/azul)
+* [Druid](https://github.com/linebender/druid) [[druid](https://crates.io/crates/druid)] — [Druid](https://linebender.org/druid/), a data-first Rust-native UI design toolkit.
```

Comment:
I'm seeing Github Actions for the CI. Can you please add a build badge for whatever of the CI jobs is the most indicative of the state of the build (as I'm seeing multiple and AFAIK Github doesn't have "everything worked" badge)